packages/sfc-playground/src/App.vue

```diff
@@ -10,7 +10,7 @@ window.addEventListener('resize', setVH)
 setVH()
 
 const useDevMode = ref(false)
-const useSSRMode = ref(false)
+const useSSRMode = ref<boolean | null>(false)
```

Comment:
I'm using `null` as a sentinel value to indicate that the SSR button shouldn't even be shown.